django/urls/converters.py

```diff
@@ -0,0 +1,91 @@
+import uuid
+
+from django.utils import lru_cache
+
+
+class BaseConverter(object):
+    pass
+
+
+class IntConverter(BaseConverter):
+    regex = '[0-9]+'
+
+    def to_python(self, value):
+        return int(value)
+
+    def to_url(self, value):
+        return str(value)
+
+
+class StringConverter(BaseConverter):
+    regex = '[^/]+'
+
+    def to_python(self, value):
+        return value
+
+    def to_url(self, value):
+        return value
+
+
+class UUIDConverter(BaseConverter):
+    regex = '[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}'
+
+    def to_python(self, value):
+        return uuid.UUID(value)
+
+    def to_url(self, value):
+        return str(value)
+
+
+class SlugConverter(BaseConverter):
+
+    # Django has both `slug_re` and `slug_unicode_re`. In the case of URLs, the
+    # non-unicode variant seems to make more sense.
```

Comment:
That is also consistent with `SlugField.allow_unicode` which defaults to `False`.